engines/wintermute/base/gfx/opengl/meshx.cpp

```diff
@@ -517,20 +520,20 @@ bool MeshX::parseTextureCoords(XFileLexer &lexer) {
 
 bool MeshX::parseNormalCoords(XFileLexer &lexer) {
 	// should be the same as _vertex count
-	uint vertexNormalCount = readInt(lexer);
-	assert(vertexNormalCount == _vertexCount);
+	uint vertexNormalCount = lexer.readInt();
+//	assert(vertexNormalCount == _vertexCount);
```

Comment:
why assert is disabled?

Reply:
The demo of "On the track of dinosaurs" has a model with fewer normals than vertices. So this condition is actually not guaranteed to be true.